test-end-to-end/android-maestro/run-maestro-test.sh

```diff
@@ -109,7 +109,8 @@ for COMMONFLOWFILE in $(ls $SCRIPTDIR/common); do
 done
 
 # Start control server
-$SCRIPTDIR/../../testserver-controller/start.sh $TESTRESULTSDIR
+#$SCRIPTDIR/../../testserver-controller/start.sh $TESTRESULTSDIR
```

Comment:
We shouldn't have commented out code in commits. If we need to know what the code was before, that should be done via git history.